email_validator/__init__.py

```diff
@@ -28,324 +29,360 @@
 
 # ease compatibility in type checking
 if sys.version_info >= (3,):
-	unicode_class = str
```

Comment:
Looks like your editor changed the indentation of every .py file you included in this patch (granted to be PEP8 compliant, but still), it should probably be reworked to match the indentation of the existing code, so reviewers can focus on the functional changes.

@JoshData, you can use http://editorconfig.org, and tell contributors to use it as well, to catch this earlier.